src/pages/studyView/studyPageHeader/studySummary/StudySummary.tsx

```diff
@@ -70,10 +74,37 @@ export default class StudySummary extends React.Component<IStudySummaryProps, {}
         return [];
     }
 
+    @action
+    updateDataDownloadable(content: boolean) {
+        this.dataDownloadable = content;
+    }
+
+    componentDidMount(): void {
+        if (this.props.studies.length === 1) {
+            request(getStudyDownloadListUrl()).then(
```

Comment:
i this will automatically fetch all study data on page load (not when user actually requests it)?